### DISCUSSION TITLE
Can't use path and fs in getStaticProps in minimum example

### DISCUSSION THREAD

**Discussion by j-holub** [vercel/next.js#21260] [Help] [answered]
Hey guys,

I'm currently experimenting a little with Next Js (I have used it in the path), but this time I have a very small project that originally I thought would only take me a few hours to complete. A cms of some sort would be overkill so I just wanted to load a few files from the harddrive, I remember that NextJs had some troubles or difficulties with that in the past, but I thougth with  `getStaticProps` that should be an easy task. Anyway, here's my code

JavaScript
```
import { path } from 'path';
import { fs } from 'fs';

export default function Index(props) {
  return (
    <h1>{props}</h1>
  )
}

export async function getStaticProps() {
  console.log(path);
  console.log(fs);
  const gameDataDir = path.join(process.cwd(), 'contents');
  const filenames = fs.readdirSync(gameDataDir);
  for (let filename of filenames) {
    console.log(filename);
  }
  return {
    props: 5
  };
}
```

This is just some experimental code to figure things out. Both `path` and `fs` are undefined. It doesn't give me the error that it simply can't find those modules, like if I had tried to use them in the browser. It does find them but they're undefined.
I've been programming for years, but this lets me feel like a total beginner again. I have no doubt, that the problem is sitting right in front of the monitor, but I've googled and tried but I can't get my head around it. Maybe a more experienced developer will see the error right away. Any help would be highly appreciated.

Node: 15.6.0 (Tried it with 14 as well)
NextJs: 10.0.5
OS: MacOS 10.15.3

**Comment by jamesmosier** [comment, marked as answer]
You should be importing `fs` and `path` like this:

```js
import path from 'path';
import fs from 'fs';
```

both are default exports and therefore do not need a named import.

**Reply by j-holub** [reply]
That did the trick, thank you. I'm sure I have tried this as well, but the other code was probably different. I gotto say, the import situation in nodejs is sometimes a little confusing and unclear. Thank you very much for the quick help.
